### ISSUE TITLE
Portal 'search' navigation links to search.dataone.org/data

### ISSUE THREAD

**Issue by gothub** [NCEAS/metacatui#1496]
Datasets viewed in a portal 'data' tab navigate back to search.dataone.org, not back to the portal 'data' tab.

1. View the Toolik portal data tab at https://search.dataone.org/portals/toolik/Data
2. click on one of the datasets in the search result listing, which for example, navigates to 
https://search.dataone.org/view/https%3A%2F%2Fpasta.lternet.edu%2Fpackage%2Fmetadata%2Feml%2Fknb-lter-arc%2F20104%2F1
3. Click on the 'Search' navigation breadcrumb

You are navigated to https://search.dataone.org, not back to the portal data tab.

**Comment by mbjones** [comment]
Good point, @gothub. I've noticed we lose track of history in a number of contexts and end up at the base search page as well.
